developer/src/kmc-ldml/test/fixtures/basic.xml

```diff
@@ -23,7 +24,8 @@
   </keys>
 
   <layers formId="us" minDeviceWidth="123">
-    <layer id="base">
+    <!-- TODO-LDML: unit test for <layers formId="us" with <layer id="base" should be illegal -->
+    <layer modifiers="none">
```

Comment:
> Use the kmxplus data rather than reading from xml directly

Does this mean the test fixture should use basic.kmx?

Reply:
No -- this is a separate e2e test for the entire ldml compiler.